lib/private/Streamer.php

```diff
@@ -126,6 +127,7 @@ public function addDirRecursive(string $dir, string $internalDir = ''): void {
 		/** @var LoggerInterface $logger */
 		$logger = \OC::$server->query(LoggerInterface::class);
 		foreach ($files as $file) {
+			if(connection_status() !== CONNECTION_NORMAL) return;
```

Comment:
```suggestion
			if(connection_aborted() === 1) {
				break;
			}
```

Reply:
I think we can even skip the `=== 1`:
```suggestion
			if (connection_aborted()) {
				break;
			}
```
Same for the following calls to `connection_status`